CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ set(BUILD_TYPE Release)
 
 set(EXECUTABLE_OUTPUT_PATH bin)
 set(LIBRARY_OUTPUT_PATH lib)
+set(INCLUDE_OUTPUT_PATH include/dso)
+set(CMAKE_OUTPUT_PATH ${LIBRARY_OUTPUT_PATH}/cmake/dso)
```

Comment:
`CMAKE_OUTPUT_PATH` seems unused. Did you have plans to include a DSOConfig.cmake DSOConfigVersion.cmake as well?

Reply:
In principle I don't plan to create those files right now but it can be done in a following step.
Are you suggesting to remove  `CMAKE_OUTPUT_PATH` in order to merge this PR?

Reply:
I was mostly just wondering why it is there, if it is unused.

Reply:
Ah! Good question. I think is because I had in mind to create a FindDSO.cmake file (sorry it was some time ago).